### DISCUSSION TITLE
Roadmap for release of 5.25 or 5.24.5

### DISCUSSION THREAD

**Discussion by Jianrong-Yu** [ivanhofer/typesafe-i18n#696] [General]
Hi @ivanhofer , may I ask is there any plan to release 5.25/5.24.5 for PR [chore: update all dependencies](https://github.com/ivanhofer/typesafe-i18n/pull/687)? We need the update on `packages/runtime/src/core.mts` to make the typesafe-i18n runtime works on Typescript `>5.1.3`.

Otherwise there are full of this error:
```
Argument of type '[]' is not assignable to parameter of type 'never'.ts(2345)
```

**Comment by ivanhofer** [comment]
Thanks for pinging me. I just have released version `5.25.0`

**Reply by Jianrong-Yu** [reply]
Thanks very much for your fast response!
